### PR TITLE
LPS-150799 [CP2.0] Add CP2NewKeyGenerationPart2B tests

### PR DESCRIPTION
**Pull request by luanakarla**

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

**Review by MathausC — COMMENTED** [review]
it looks ok to me.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPart2B.testcase [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+				value = "Liferay Test");
+
+			CP2ActivationKeys.environmentDetails(
+				environmentDetails = "name",
+				value = "");
+
+			KeyPress(
+				locator1 = "//body",
+				value1 = "\ESCAPE");
+
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPart2B.testcase:187]: I couldn't find a generic path for this, but I created one. Tell me if it is ok?

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:relevant

**Comment by luanakarla** [comment]
ci:stop

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:stable

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:stable

**Comment by luanakarla** [comment]
ci:test:stable

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPart2B.testcase (hunk 1 above)
[the remark this answers is not in the document]
Comment (barrosdennys) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPart2B.testcase]: Hi @luanakarla there is this one:
KaleoDesignerEditWorkflow#BODY

but that's OK. I will fw it.

**Comment by barrosdennys** [comment]
ci:forward

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:forward

**Comment by luanakarla** [comment]
ci:close

**Pull request closed without merging**